api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java

```diff
@@ -40,7 +40,11 @@
     @Minimum(1)
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public Integer getNumStreams() {
-        return numStreams;
+        if (numStreams == null) {
```

Comment:
I don't think we need this. I guess that you did that for filling the related env var for the underlying Docker image. Btw the `numStreams` and the related env var are for the `--numstreams` option for the mirror maker command line tool which is not mandatory. It means that a not set env var means just no `--numstreams` so just 1 as default. I think that you should just not pass the env var (so that it's empty) when `numStreams' is 'null` in the MMO (also for this reason it's an `Integer` and not a primitive `int`). Or maybe if you pass `null`, the fabric8 client doesn't create the env var object.